UKHomeOffice/cerberus-service
# COP-9118 Update webpack

## Description
Moderate and High errors in packages that require latest webpack-cli and webpack-dev-service of at least 4.3.1

Updated

## To Test

pull down
run npm i
start cerberus ui
> it should build and run on localhost:8080

## Developer Checklist

\* Required
